reposerver/repository/app_version.go

```diff
@@ -56,19 +56,16 @@ func getVersionFromFile(appPath, jsonPathExpression string) (*string, error) {
 		return nil, fmt.Errorf("Unsupported file format of %s", appPath)
 	}
 
-	jp := jsonpath.New("jsonpathParser")
-	jp.AllowMissingKeys(true)
-	if err := jp.Parse(jsonPathExpression); err != nil {
-		return nil, err
-	}
-
-	var buf strings.Builder
-	err = jp.Execute(&buf, jsonObj)
+	versionValue, err := jsonpath.Get(jsonPathExpression, jsonObj)
 	if err != nil {
 		return nil, err
 	}
+	appVersion, ok := versionValue.(string)
+	if !ok {
+		log.Infof("Version value is not a string")
```

Comment:
let's add more version config details to this log to make troubleshooting easier